packages/extension/src/companion/companionRequest.tsx

```diff
@@ -8,7 +8,7 @@ const proxyRequest = {
       url: args?.[0],
       document: args?.[1],
       variables: args?.[2],
-      queryKey: args?.[3],
+      headers: args?.[3],
```

Comment:
Why did we decide to use headers instead of queryKey?

Reply:
I felt like it was much more fitting to use the headers, and that the third parameter was the header part - not sure why my initial implementation didn't throw an error but the other one did.